Containerfile

```diff
@@ -0,0 +1,51 @@
+# This file will contain the configuration for containerized esi-leap
+
+# Use python as the parent image for esi-leap
+
+FROM python:3.11
+
+# Use label descriptor
+
+LABEL description="ESI-LEAP container file"
```

Comment:
I'd probably change "container file" to "container image" here, since `LABEL` describes the image being created.